sections/03_categorize.md

```diff
@@ -167,6 +167,35 @@ advances to practical game-changers.
 Additionally, unique barriers exist in this space that may hinder progress in
 this field.
 
+###### Lack of ground-truth labels in data, and high cost in post-study validation by clinician experts.
```

Comment:
Is 'pyrite standard' a common term?  I understand what you mean and it's clever if you coined it, but I had to think about it for a second or two.  We might need to explain the phrase if we use it.  I've also heard this (imperfect 'gold standard') called a 'bronze standard'.